app/serializers/resource_serializer.rb

```diff
@@ -1,10 +1,20 @@
 # frozen_string_literal: true
 
 class ResourceSerializer < ActiveModel::Serializer
+  type 'resource'
   attributes :id, :name, :abbreviation, :onesky_project_id
 
   belongs_to :system
 
   has_many :translations
   has_many :pages
+
+  has_many :latest_translations
+
+  def latest_translations
+    Translation.find_by_sql("select T.* from translations T inner join (select language_id, resource_id, max(version) as
```

Comment:
A few things here.
1. Are you sure you need raw SQL for this? It is unclear what exactly this does.
2. If you do need it, consider moving it to an instance method on the Resource class.
3. Are you sure rubocop is running on this? I'm pretty sure this is not the best way to do a block of text, so I am surprised you got this past it.